src/modules/auth/dto/auth.dto.ts

```diff
@@ -0,0 +1,28 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { AuthDtoConstants } from 'src/utils/constants/docsTexts';
+
+export class UserDto {
+  @ApiProperty({
+    description: AuthDtoConstants.id.description,
```

Comment:
its better here with the text right away, as its something that developer use, and its will be uncomfortable to check each constant to all examples and descriptions